E1b-ARCT/ALCT1/Dataset/README.md

```diff
@@ -0,0 +1,141 @@
+## Variations
+The following occurrence variations need to be checked and certified in relation to the targeted entities and concept templates:
+
+- IfcAlignmentHorizontalSegment - *includes LINE, CIRCULARARC and CLOTHOID*
+- IfcAlignmentVerticalSegment - *includes CONSTANTGRADIENT and CIRCULARARC*
+- IfcAlignmentCantSegment - *includes CONSTANTCANT and LINEARTRANSITION*
+
+## Entities Table
+
+- IfcProject
+- IfcOwnerHistory
+- IfcPersonAndOrganization
+- IfcPerson
+- IfcOrganization
+- IfcApplication
+- IfcDirection
+- IfcCartesianPoint
+- IfcAxis2Placement3D
+- IfcGeometricRepresentationContext
+- IfcSIUnit
+- IfcUnitAssignment
+- IfcSite
+- IfcLocalPlacement
+- IfcRelAggregates
+- IfcRelContainedInSpatialStructure
+- IfcAlignment
+- IfcAlignmentHorizontal
+- IfcAlignmentVertical
+- IfcAlignmentCant
+- IfcAlignmentSegment
+- IfcAlignmentHorizontalSegment
+- IfcAlignmentVerticalSegment
+- IfcAlignmentCantSegment
+- IfcRelNests
+- IfcCompositeCurve
+- IfcLine
+- IfcCircle
+- IfcClothoid
+- IfcGradientCurve
+- IfcSegmentedReferenceCurve
+- IfcCurveSegment
+- IfcProductDefinitionShape
+- IfcShapeRepresentation
+- IfcVector
+- IfcAxis2Placement2D
+
+## Model Dataset
+This test case utilises the attached dataset documented by the following drawings and data schedule. 
+
+
+#### Horizontal Segments
+| start_point_x | start_point_y | start_direction | start_radius_of_curvature | end_radius_of_curvature | segment_length | gravity_center_line_height | predefined_type |
+| ------------- | ------------- | ------------- | ------------- | ------------- | ------------- | ------------- | ------------- |
```

Comment:
I'd prefer to also see the units of each column somewhere. Applies to all three tables.

Reply:
Units are added